### PR TITLE
Person Pro role

### PR DESCRIPTION
**Pull request by mllocs**

### CODE REVIEW

File: engines/donalo/app/services/donalo/person_role.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
+module Donalo
+  class PersonRole
+    attr_reader :person
+
+    CUSTOM_FIELD_NAME = 'Rol'.freeze
```
Comment (mllocs) [inline engines/donalo/app/services/donalo/person_role.rb]: Should be "_Rol" 🔍

File: engines/donalo/app/services/donalo/person_role.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    end
+
+    def pro?
+      return false unless person
+      return false unless pro_option_id
+
+      custom_field_value_options.include? pro_option_id
+    end
+
+    def custom_field_value_options
```
Comment (mllocs) [inline engines/donalo/app/services/donalo/person_role.rb:21]: make `private` 👇

**Comment by sseerrggii** [comment]
:+1: 

How can test this on staging @sauloperez ?

**Comment by sauloperez** [comment]
@mllocs I merged master into this one so we get `3414a4a43`, which fixed the issue with mimemagic by upgrading. Otherwise, deployments fail to install gems.

**Comment by sauloperez** [comment]
@sseerrggii I just deployed to https://donalo-staging.herokuapp.com/. You'll find the credentials for the two demo users in BW.

**Comment by sseerrggii** [comment]
@mllocs could you review that this feature is on staging? I'm editing user but I don't find the pro_rol field

**Comment by sauloperez** [comment]
fields need a DB migration but I'm afraid this is not yet finished @sseerrggii

**Comment by sseerrggii** [comment]
For me i'ts ok, @sauloperez we can deploy it to production

**Review by sauloperez — APPROVED** [review]

**Comment by sauloperez** [comment]
Deployed to production :tada:

**Pull request merged**